### PR TITLE
ST-385-spoke-test  [Do not merge]

### PR DESCRIPTION
**Pull request by geciciserkan**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ST-385

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by muradalamv1 — APPROVED** [review]

**Review by sjamihmcts — APPROVED** [review]

**Review by spotstory — CHANGES_REQUESTED** [review]
Just in case, can we disable our deployment to AAT for the time being - I don't think we should risk an AAT deployment with the Jurisdiction still set to DIVORCE

File: src/functionalTest/java/uk/gov/hmcts/sptribs/testutil/DocumentManagementStore.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            MediaType.APPLICATION_PDF_VALUE,
+            resourceAsBytes(filePath)
+        );
+
+        final String accessToken = idamTokenGenerator.generateIdamTokenForSystem();
+        final String userId = idamTokenGenerator.getUserDetailsFor(accessToken).getId();
+
+        final UploadResponse uploadResponse = documentUploadClientApi.upload(
+            accessToken,
+            serviceAuthenticationGenerator.generate("nfdiv_case_api"),
```
Comment (osheaf01) [inline src/functionalTest/java/uk/gov/hmcts/sptribs/testutil/DocumentManagementStore.java:49]: Does this need to be "nfdiv_case_api"? 
Extract out as a property?

**Review by osheaf01 — DISMISSED** [review]
LGTM, just one question inline

**Review by muradalamv1 — APPROVED** [review]

**Review by muradalamv1 — APPROVED** [review]

**Review by jaberelfasi — APPROVED** [review]

**Comment by geciciserkan** [comment]
> Just in case, can we disable our deployment to AAT for the time being - I don't think we should risk an AAT deployment with the Jurisdiction still set to DIVORCE

**Review by spotstory — APPROVED** [review]

**Pull request merged**